iac-extensions/spring-config/src/test/java/org/sonar/iac/springconfig/SpringConfigAssertions.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.springconfig;
+
+import javax.annotation.Nullable;
+import org.sonar.api.batch.sensor.issue.ExternalIssue;
+import org.sonar.iac.common.api.tree.impl.TextRange;
+import org.sonar.iac.common.testing.ExternalIssueAssert;
+import org.sonar.iac.common.testing.TextRangeAssert;
+import org.sonar.iac.common.testing.TextRangePluginApiAssert;
+
+public class SpringConfigAssertions {
+  public static TextRangeAssert assertThat(@Nullable TextRange actual) {
+    return TextRangeAssert.assertThat(actual);
+  }
+
+  public static TextRangePluginApiAssert assertThat(@Nullable org.sonar.api.batch.fs.TextRange actual) {
+    return TextRangePluginApiAssert.assertThat(actual);
+  }
+
+  public static ExternalIssueAssert assertThat(@Nullable ExternalIssue issue) {
+    return ExternalIssueAssert.assertThat(issue);
+  }
```

Comment:
What is the benefit of wrapping the calls again? For me it creates yet another layer to use / forget.

Reply:
I feel like, most of the time only the TextRangeAssert is needed.

Reply:
Yes, mostly `TextRangeAssert` is needed because we are creating another assertions when we see the benefit from it. See `DockerAssertions` it contains more useful assertions.
The befit is to have all assertions for the module in one place. It is recommended by AssertJ: https://assertj.github.io/doc/#assertj-core-custom-assertions-entry-point